tests/scanbuild_py_makefile.txt

```diff
@@ -0,0 +1,12 @@
+# Test scan-build-py
```

Comment:
Maybe add a word about scan-build-py not being able to instrument raw calls to clang?

Reply:
Documented!